helm/cluster-vsphere/values.yaml

```diff
@@ -1,134 +1,268 @@
-# cluster
-# Cluster-specific configuration
-cluster:
-
-  # cluster.kubernetesVersion
-  # Kubernetes version to deploy. Must match the version available in the
-  # image defined at template.templateName
-  kubernetesVersion: ""
-
-  # cluster.name
-  # Cluster name; used as a base for names of all cluster resources.
-  name: ""
+apiServer:
+  enableAdmissionPlugins: "NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota,PersistentVolumeClaimResize,DefaultStorageClass,Priority,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook"
+  featureGates: "TTLAfterFinished=true"
+controllerManager:
+  featureGates: "ExpandPersistentVolumes=true,TTLAfterFinished=true"
 
-  # cluster.organization
-  # The organization which owns this cluster.
-  organization: ""
+clusterDescription: ""  # Cluster description used in metadata.
 
-# network
-# Network-specific configuration options.
+servicePriority: "highest"
+clusterLabels: {}
 network:
+  podsCidrBlocks:
+  - "192.168.0.0/16"
+  servicesCidrBlocks:
+  - "192.168.1.0/16"
+  controlPlaneEndpoint:
+    host: ""  # [string] Manually select an IP for kube API. "" for auto selection from pool.
+    port: ""  # [int] Port to use. "" defaults to 6443.
+
+
+controlPlane:  # Must match nodeClasses' fields except "name" and must contain "replicas".
+  replicas: 0  # Number of control plane instances to create (odd number).
+  machineTemplate:  # check .nodeClasses.default for docs
+    template: ""  # template used to create (or) upgrade control plane nodes.
+    resourcePool: ""  # ResourcePool is the name or inventory path of the resource pool in which the virtual machine is created/located.
+    memoryMiB: 8192
+    network:
+      devices:
+      - dhcp4: true
+        networkName: 'grasshopper-capv'
+  etcd:
+    imageRepository: "giantswarm"
+    imageTag: 3.5.4-0-k8s
+  dns:
+    imageRepository: projects.registry.vmware.com/tkg
+    imageTag: v1.7.0_vmware.12
+  image:
+    repository: projects.registry.vmware.com/tkg
+  resourceRatio: 8  # Ratio between node resources and apiserver resource requests.
+
+nodePools:
+  worker:
+    class: default
+    replicas: 2
+
+ssh:
```

Comment:
Please mimic this interface https://github.com/giantswarm/cluster-cloud-director/blob/main/helm/cluster-cloud-director/values.yaml#L98